test/functional/apps/visualize/index.ts

```diff
@@ -37,6 +39,7 @@ export default function({ getService, loadTestFile }: FtrProviderContext) {
         defaultIndex: 'logstash-*',
         'format:bytes:defaultPattern': '0,0.[000]b',
       });
+      isOss = await PageObjects.common.isOss();
```

Comment:
This feels like a type of change that would be best in a separate PR

Reply:
It could be.  It isn't directly related to this change.  It's a bug that passes in CI (OSS/OSS) but fails when run against the default build including Cloud.

Reply:
will pull out the changes in a separate PR.

Reply:
pulled out the changes to a separate PR: https://github.com/elastic/kibana/pull/60134